docs/source/conf.py

```diff
@@ -175,7 +175,7 @@
 
 # Useful aliases to avoid repeating long URLs.
 extlinks = {'github-demo': (
-    'https://github.com/enthought/traitsui/tree/master/traitsui/examples/demo/%s',
+    'https://github.com/enthought/traitsui/tree/main/traitsui/examples/demo/%s',
```

Comment:
I'll fix this in a separate PR - we shouldn't be linking to the example of `main` branch. Instead, I think we should be linking to the specific version/tag.